torch/csrc/jit/codegen/cuda/kernel.cpp

```diff
@@ -82,6 +91,7 @@ void Kernel::analyze() {
   const KernelIrScanner ir_scanner(exprs_);
```

Comment:
@rdspring1, would you mind adding a perf scope here too? (`FUSER_PERF_SCOPE("Kernel::analyze");`)

Reply:
:+1: